Framework/CurveFitting/src/Algorithms/Fit.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "MantidAPI/CostFunctionFactory.h"
 #include "MantidAPI/FuncMinimizerFactory.h"
+#include "MantidAPI/FunctionDomain1D.h"
```

Comment:
No. I forgot to remove it after I moved the peak radius stuff to the base class.